src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocmissingleadingasterisk/Example1.java

```diff
@@ -14,34 +15,36 @@
  *   int value = 0;
  * </pre>
  */
-class JavaStyle {} // ok
+class JavaStyle {}
 
 /** Valid Scala-style comment.
   * Some description here.
   **/
-class ScalaStyle {} // ok
+class ScalaStyle {}
 
 /** **
   * Asterisks on first and last lines are optional.
   * */
-class Asterisks {} // ok
+class Asterisks {}
 
 /** No asterisks are required for single-line comments. */
-class SingleLine {} // ok
+class SingleLine {}
 
-/** // violation on next blank line, javadoc has lines without leading asterisk.
+// violation on next blank line, javadoc has lines without leading asterisk.
+/** // violation below
```

Comment:
can we remvoe `// violation below` ?
we should have only one comment that points on violation that such comment should be outside of javadoc comment.

please apply to all